src/databricks/labs/ucx/assessment/azure.py

```diff
@@ -534,3 +541,7 @@ def _get_storage_accounts(self) -> list[str]:
                 if storage_acct not in storage_accounts:
                     storage_accounts.append(storage_acct)
         return storage_accounts
+
+    @property
+    def filename(self):
```

Comment:
Why do we need to expose filename to public? Keep all read/write operations with this file to this class.